cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -97,6 +97,7 @@
     {# Customized Modernizr build that includes html5shiv.
        Built via gulp-modernizer in `scripts.js` task. #}
     <script src="{{ static('js/modernizr.min.js') }}"></script>
+    <script src="{{ static('js/es5-shim.js') }}"></script>
```

Comment:
This doesn't have to go in base.html. This could be added as the first script in common.js.